k8s-openapi-codegen-common/src/templates/impl_deserialize.rs

```diff
@@ -80,14 +82,14 @@ pub(crate) fn generate(
 					r#"                        Field::Key_{} => value_{} = Some(serde::de::MapAccess::next_value(&mut map)?),"#,
 					field_name, field_name)?;
 
-				writeln!(field_value_assignment, "                    {}: value_{}.ok_or_else(|| serde::de::Error::missing_field({:?}))?,", field_name, field_name, name)?;
+				writeln!(field_value_assignment, "                    {}: value_{}.ok_or_else(|| serde::de::Error::missing_field({:?}))?,", safe_field(field_name), field_name, name)?;
```

Comment:
`field_name` should already be sanitized before this function is called. That's why it exists in addition to `name`.

Reply:
The is the same issue as above. `field_name` is being used as "field name" and then as "a part of a field name".

As mentioned in the other comment, I went ahead and merged this. But I still think it would be nice to use Rust's raw identifiers in cases like that.